frontend/sozisel-app/src/components/utils/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import "./button.scss";
+
+import { ReactElement } from "react";
+
+interface ButtonProps {
+  name: string;
+  type: "button" | "submit" | "reset" | undefined;
+}
+
+export default function Button({ name, type }: ButtonProps): ReactElement {
```

Comment:
To be honest I'm not sure right now about our own components. You don't even extend button props via this component which results in discarding all of its functionalities. If you don't know how to come up with proper button interface we should probably give up and use MaterialUI or Chakra.